src/vm/eetwain.cpp

```diff
@@ -5913,6 +5913,14 @@ ULONG32 EECodeManager::GetStackParameterSize(EECodeInfo * pCodeInfo)
     } CONTRACTL_END;
 
 #if defined(_TARGET_X86_)
+#if defined(WIN64EXCEPTIONS)
+    if (pCodeInfo->IsFunclet())
```

Comment:
As I understand, it depends on how JITer treats funclet's info. It seems that JIter usually do not distinguish function and funclet in terms of frame layout as far as I know.